pandas/tests/series/test_analytics.py

```diff
@@ -1330,6 +1330,12 @@ def test_duplicate_keep_all_ties(self):
         expected = Series([6, 7, 7, 7, 7], index=[7, 3, 4, 5, 6])
         assert_series_equal(result, expected)
 
+    def test_boolean(self):
+        s = Series([True, False])
+        result = s.nlargest(1)
+        expected = s.sort_values(ascending=False).head(1)
```

Comment:
Let's explicitly construct `expected` with a `Series` call instead of leveraging `sort_values`.